GoogleForCreators/web-stories-wp
# Edit Media Dialog: Use Text Area for Video Description

## Context


When editing an uploaded video in the editor, the description can get cut off if it is really long. 

## Summary


When selecting the ellipsis to edit media from the editor a `Dialog` pops up.  We want to use a text area as opposed to an input element. 

## Relevant Technical Choices


~The `Dialog` component in `Design Systems` didn't have a way to override the `DialogBox`'s styles. Added a prop to allow custom styling of the dialog box. If the file name is 21 characters or more, the dialog box is now given 100% max-width. This allows for approximately a minimum of 49 characters to be displayed before needing to scroll. This is due to the content below.  I could make `max-width` determined by `vw`, but 49-ish characters seems enough, thoughts?~

Above changes are obsolete. 

Since it is likely a video description be more than a few short words, I've updated the component to use `TextArea` as opposed to `Input` for this.  

## To-do



## User-facing changes


<img width="760" alt="Screen Shot 2021-12-06 at 2 45 02 PM" src="https://user-images.githubusercontent.com/1820266/144927706-60fb77e6-6202-4b13-96bf-099a4e05e08f.png">
<img width="882" alt="Screen Shot 2021-12-06 at 2 44 40 PM" src="https://user-images.githubusercontent.com/1820266/144927710-db0f7c23-fffb-4f51-80d9-9f60bfda5062.png">



## Testing Instructions






This PR can be tested by following these steps:

1. upload an image with a very long name. 
2. In the editor select the `...` on a video
3. Select 'edit' from the menu
4. You should see a Modal appear and the  video description should now be a text area box instead of a single line input. 

## Reviews

### Does this PR have a security-related impact?
no


### Does this PR change what data or activity we track or use?
no


### Does this PR have a legal-related impact?
no


## Checklist




---



See #4576
